### PR TITLE
SONARIAC-833 Add Azure Resource Manager Documentation

### PR DESCRIPTION
**Pull request by mstachniuk**

### CODE REVIEW

File: sonar-iac-plugin/src/main/resources/static/azureresourcemanager_doc.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: AzureResourceManager
+key: azureresourcemanager
+---
+
+## Supported Versions
+
+| Format | Version    | Status              |
+|--------|------------|---------------------|
+| JSON   | 2023-06-01 | Partially Supported |
```
Comment (mstachniuk) [inline sonar-iac-plugin/src/main/resources/static/azureresourcemanager_doc.md:10]: It is not clear to me what version of JSON format should we put here. I choose the newest stable version from here: https://github.com/Azure/azure-resource-manager-schemas/tree/main/schemas Feel free to challenge it.

File: sonar-iac-plugin/src/main/resources/static/azureresourcemanager_doc.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+| Bicep  | any        | Not Supported       |
+
+## Language-Specific Properties
+
+Discover and update the Azure Resource Manager [properties](/analysis/analysis-parameters/) in: **<!-- sonarcloud -->Project <!-- /sonarcloud -->[Administration > General Settings > Languages > AzureResourceManager](/#sonarqube-admin#/admin/settings?category=AzureResourceManager)**
```
Comment (mstachniuk) [inline sonar-iac-plugin/src/main/resources/static/azureresourcemanager_doc.md:15]: @nils-werner-sonarsource Could you validate this links, please? I don't have admin access.

**Review by nils-werner-sonarsource — APPROVED** [review]
LGTM! I'm fine with the version for the JSON schema

File: sonar-iac-plugin/src/main/resources/static/azureresourcemanager_doc.md (hunk 1 above)
Comment (nils-werner-sonarsource) [reply sonar-iac-plugin/src/main/resources/static/azureresourcemanager_doc.md:10]: maybe we can say `up to` since we are not aware of any problems.

**Pull request merged**